kevm-pyk/src/kevm_pyk/kevm.py

```diff
@@ -699,4 +699,16 @@ def _process_jumpdests(bytecode: bytes, offset: int) -> list[KToken]:
     :return:  A list of intToken instances representing the positions of all found jump destinations in the bytecode adjusted by the offset.
     :rtype: list[KToken]
     """
-    return [intToken(offset + i) for i, byte in enumerate(bytecode) if byte == 0x5B]
+    push1 = 0x60
+    push32 = 0x7F
+    jumpdest = 0x5B
+    i = 0
+    jumpdests: list[KToken] = []
+    while i < len(bytecode):
+        if push1 <= bytecode[i] <= push32:
+            i += bytecode[i] - push1 + 2
+            continue
+        if bytecode[i] == jumpdest:
+            jumpdests.append(intToken(offset + i))
+        i += 1
```

Comment:
```suggestion
        if push1 <= bytecode[i] <= push32:
            i += bytecode[i] - push1 + 2
            continue
        else:
            if bytecode[i] == jumpdest:
                jumpdests.append(intToken(offset + i))
            i += 1
```

Perhaps like this instead? Whichever reads better to you.

Reply:
Done.